src/main/java/frc/robot/commands/vision/LimelightAlign.java

```diff
@@ -32,7 +33,12 @@
 
     private final long timeout;
     private long startTime = 0;
-    RollingAverage rollingAverageRCW, rollingAverageSTR, rollingAverageFWD  = new RollingAverage(Const.RollingAverage.WINDOW_SIZE);
+    RollingAverage rollingAverageRCW, rollingAverageSTR, rollingAverageFWD  = new RollingAverage(rollingAverageWindow);
```

Comment:
Where is `rollingAverageWindow`?

Also, I can take this as a teaching moment. This may look like it assigns the same value to all three variables; but it does not: only `rollingAverageFWD` gets assigned the new object. This is a rookie mistake that Ethan (@snyderethan) made. I didn't even realize he was doing this.

Reply:
@jordanbancino But it just looks so neat